bayesian/api/api_v2.py

```diff
@@ -241,7 +247,7 @@ def post():
         # Step4: Handle Unknown Packages
         if unknown_pkgs:
             stack_recommendation = add_unknown_pkg_info(stack_recommendation, unknown_pkgs)
-            if os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW") != "1" and ecosystem != "golang":
+            if os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW") != "1":
```

Comment:
Can you do the same here as backbone? i.e. letting the jobs to decide whether to ingest based on environment(staging?)